Tests/KituraNetTests/ClientE2ETests.swift

```diff
@@ -335,4 +337,19 @@ class ClientE2ETests: KituraNetTest {
             }
         }
     }
+
+    class TestSimpleClientDelegate: ServerDelegate {
+
+        func handle(request: ServerRequest, response: ServerResponse) {
+            do {
+                let result: String = "Hello, World!"
+                response.statusCode = .OK
+                response.headers["Content-Type"] = ["text/plain"]
+                response.headers["Content-Length"] = ["\(result.count)"]
+                try response.end(text: result)
+            } catch {
+                print("Error while writing response")
```

Comment:
Can we have an `XCTFail(...)` here instead of the `print(...)`? Its better to fail early.

Reply:
Sure @pushkarnk